stopcovid/status/drill_instances.py

```diff
@@ -169,6 +172,20 @@ def save_drill_instance(self, drill_instance: DrillInstance):
                 .values(**settings)
             )
 
+    def get_incomplete_drills(self, inactive_for_minutes=None) -> List[DrillInstance]:
+        stmt = select([drill_instances]).where(
+            drill_instances.c.completion_time == None
+        )  # noqa:  E711
+        if inactive_for_minutes is not None:
+            stmt = stmt.where(
+                drill_instances.c.current_prompt_start_time
+                >= datetime.datetime.now(datetime.timezone.utc)
+                - datetime.timedelta(minutes=-1 * inactive_for_minutes)
```

Comment:
- should it be <= and not >=?
- Is the double negative a bug? `-1 *` seems like a mistake as you are already subtracting.

Reply:
You are definitely right